mock/cpp_mock_scenarios/src/follow_trajectory/follow_polyline_trajectory_with_do_nothing_plugin.cpp

```diff
@@ -0,0 +1,133 @@
+// Copyright 2015 TIER IV, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <quaternion_operation/quaternion_operation.h>
+
+#include <ament_index_cpp/get_package_share_directory.hpp>
+#include <cpp_mock_scenarios/catalogs.hpp>
+#include <cpp_mock_scenarios/cpp_scenario_node.hpp>
+#include <rclcpp/rclcpp.hpp>
+#include <traffic_simulator/api/api.hpp>
+#include <traffic_simulator_msgs/msg/behavior_parameter.hpp>
+#include <traffic_simulator_msgs/msg/polyline.hpp>
+#include <traffic_simulator_msgs/msg/vertex.hpp>
+
+// headers in STL
+#include <memory>
+#include <string>
+#include <vector>
```

Comment:
The term `STL` is not a term that exists in the C++ standard. Some of the standard libraries are still conventionally called STL, but at least some of the libraries you call "STL" definitely include `memory` that is not included in STL, even by convention.If you

If you think that "STL" is a term referring to the C++ standard library, you are wrong. Please do not use this terminology.

Reply:
Fixed in 17e1aa657fea28861cb18583b7579189bce10c29